integrations/helm/release/release.go

```diff
@@ -189,12 +189,15 @@ func (r *Release) Install(chartPath, releaseName string, fhr flux_v1beta1.HelmRe
 
 		if err != nil {
 			r.logger.Log("error", fmt.Sprintf("Chart release failed: %s: %#v", releaseName, err))
-			// if an install fails, purge the release and keep retrying
-			r.logger.Log("info", fmt.Sprintf("Deleting failed release: [%s]", releaseName))
-			_, err = r.HelmClient.DeleteRelease(releaseName, k8shelm.DeletePurge(true))
-			if err != nil {
-				r.logger.Log("error", fmt.Sprintf("Release deletion error: %#v", err))
-				return nil, err
+			// purge the release if the install failed but only if this is the first revision
```

Comment:
This happening is weird as this _is_ already in master.

Reply:
It is a little odd. Maybe check locally what happens when you do this merge,

Reply:
I think GitHub is tricking us

```
➜ git checkout master            
Switched to branch 'master'

flux on  master [$] via 🐹 v1.11.2 at ☸️  <snip> 
➜ git merge release/helm-0.5.x  
Merge made by the 'recursive' strategy.
 CHANGELOG-helmop.md                       | 15 +++++++++++++++
 deploy-helm/helm-operator-deployment.yaml |  2 +-
 2 files changed, 16 insertions(+), 1 deletion(-)
```

Reply:
Tricksy GitHub! May as well push that merge, to close the PR.